crypto/rand/drbg_lib.c

```diff
@@ -82,6 +82,12 @@ static unsigned int slave_reseed_interval  = SLAVE_RESEED_INTERVAL;
 static time_t master_reseed_time_interval = MASTER_RESEED_TIME_INTERVAL;
 static time_t slave_reseed_time_interval  = SLAVE_RESEED_TIME_INTERVAL;
 
+/* A logical OR of all used DRBG flag bits (currently there is only one) */
+static const unsigned int rand_drbg_used_flags =
+    RAND_DRBG_FLAG_CTR_NO_DF |
+    /* add new entries before this line */
+    0;
```

Comment:
These two lines should go I think.

i.e. either
```
static const unsigned int rand_drbg_used_flags =
    RAND_DRBG_FLAG_CTR_NO_DF ;
```
or everything on one line (if it fits);
Future items being added with the `|` leading the line.

Reply:
The original intention was to be diff-friendly, i.e., adding one flag will produce only a single line diff. But I understand that it violates the coding style guidelines, so I'll change it.

Reply:
Diffs don't handle things that move across files well.
Still approved.